main.nginx.conf

```diff
@@ -84,7 +84,7 @@ http {
       # Offline new Submissions - can be public if it has st query parameter
       return 301 "/f/$enketoId/offline$is_args$args";
     }
-    location ~ "^/-/(?<enketoId>[a-zA-Z0-9]+)$" {
+    location ~ "^/-(?!thanks$)(?<enketoId>[a-zA-Z0-9]+)$" {
```

Comment:
because we removed length of `enketoId` from here so now it is matching `thanks` as well.

Updated regex to excluding 'thanks' from matching.